tests/ui/test_login.py

```diff
@@ -1,11 +1,25 @@
+import os
 import pytest
 
 from pages.desktop.home import Home
+from seleniumlogin import force_login
```

Comment:
Fxa and ```live_server``` don't play well. Spent a few days trying to debug with @mstriemer but didn't get much.